### ISSUE TITLE
Change default mapping from bowtie2 to minimap2

### ISSUE THREAD

**Issue by jianshu93** [bxlab/metaWRAP#342]
Bowtie2 does not scale well with increase threads but minimap2 does and is much faster than bowtie2 but the results are nearly identical after a few testing. I think this will greatly improve the speed for binning and also minimap2 support gzipped fastq file very well, which means it is easy to allow support for gzipped fast file. I can be a contributor if you like. Also calculation of relative using salmon is not a very good choice. I think coverM is the best tool and it can also calculate the gene RPKM et.al. I will be willing to incorporate them into this software if you welcome. Thanks, Jianshu

**Comment by ursky** [comment]
I completely agree that a few steps are somewhat out of date, its just something that happens with bioinformatics software after a few years. However, they still provide accurate and valid results with minimal user involvement. Because I have a full time job and cant devote much time I am basically only doing maintanance and bug fixing now, not new feature additions. If I (or someone else) ever make MetaWrap2, I will make sure it has the more up-to date methods.

**Comment by harish0201** [comment]
@jianshu93 Would it be possible to have some benchmarks? I was wondering about using minimap2 over bowtie2 as well!

**Comment by jianshu93** [comment]
Yes there are some. Both bwa mem2 and minimap2 are much faster than bowtie2 with nearly identical coverage depth.

Jianshu
